core-bundle/src/Messenger/Transport/CronFallbackTransport.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Messenger\Transport;
+
+use Contao\CoreBundle\Cron\Cron;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Transport\TransportInterface;
+
+class CronFallbackTransport implements TransportInterface
```

Comment:
This is the one that simply checks if we have a real cron job for every call. If so, we can use the configured target (Doctrine by default), if not -> fallback (sync).